### ISSUE TITLE
Rhubarb issue

### ISSUE THREAD

**Issue by cutydina** [opentoonz/opentoonz#5064]
### Description

hello! don't know if this will be an OpenToonz issue or a Rhubarb issue, just want to know if it's something i can fix on OpenToonz?

```
Error processing file C:Users\USER\AppData\Local\OpenToonz\cache\rhubarb\temp.wav 
Error performing speech recognition via PoketSphinx tools.
Could not determine path of bin directory.
```

### Steps to Reproduce

1. Use rhubarb on OpenToonz. Sometimes, not always.
2. 

### Expected Behavior

Do a lypsync syncronization.

### Screenshots, Video & Crash Logs

![Captura de pantalla 2023-08-25 194800](https://github.com/opentoonz/opentoonz/assets/103970174/0ec2c77b-2918-417e-9657-fd8d8f0bdb21)

### OpenToonz Version

Nightly (Specify below)

### OpenToonz Version Information

24/august

### Operating System

Windows

### GPU

NVIDIA

### Graphics Tablet

Huion

**Comment by RodneyBaker** [comment]
Closing as likely resolved and/or abandoned.

**Comment by ChrisLora** [comment]
Did you ever figure this out? I'm having the same issue here. I tried all the basics: uninstall/reinstall, same for ffmpeg and rubharb files, did a full restart. Tested on old files with old audio I've successfully used before and even those don't work now. So idk, the mention of the "bin" repository to me seems important but the only bin I'm aware of is the one containing the ffmpeg files but that can't possibly be the issue
